installation/scripts/testing.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
```

Comment:
Please use kyma CLI:
Here are the sources, so you can see which flags are available: https://github.com/kyma-project/cli/blob/master/pkg/kyma/cmd/test.go

Reply:
Okay, now I know that we can't use it because it calls helm test underneath. Please create GitHub issue for Kyma CLI about that.